front/pages/forgot_password.tsx

```diff
@@ -0,0 +1,106 @@
+import { useState } from 'react';
+import { Formik, Form } from 'formik';
+import { makeStyles } from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
+import { useForgot } from 'hooks/forgot_password';
+
+import withApollo from 'utils/withApollo';
+import TextInput from 'components/TextInput';
+import Button from 'components/Button';
+import Head from 'components/Head';
+import Logo from 'components/Logo';
+import { ForgotParams, forgotSchema } from 'utils/validation';
+
+const useStyles = makeStyles((theme) => ({
+  text: {
+    display: 'flex',
+    justifyContent: 'center',
+    marginTop: theme.spacing(3),
+    marginBottom: theme.spacing(5),
+    minWidth: 175,
+    fontSize: 24,
+  },
+  logoContainer: {
+    display: 'flex',
+    height: 'auto',
+    paddingTop: '3vw',
+    justifyContent: 'center',
+    alignItems: 'center',
+    [theme.breakpoints.down('sm')]: {
+      paddingTop: '8vw',
+    },
+  },
+  form: {
+    display: 'flex',
+    width: '20%',
+    margin: 'auto',
```

Comment:
you shouldn't use `margin: auto` to center your content, try `alignItems: 'center'` or `justifyContent: 'center'`